app/views/curation_concerns/base/_representative_media.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if presenter.representative_id.present? && (fs = ::FileSet.load_instance_from_solr(presenter.representative_id)) %>
+<% if presenter.representative_id.present? && (fs = ::FileSet.find(presenter.representative_id)) %>
```

Comment:
@jcoyne @mjgiarlo I'm inclined to ticket this and mark it for 7.0.0. We could then get a beta out this week.